hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/auth/ITestAssumeRole.java

```diff
@@ -255,8 +255,7 @@ public void testAssumeRoleBadInnerAuth() throws Exception {
     conf.set(SECRET_KEY, "not secret");
     expectFileSystemCreateFailure(conf,
         AWSBadRequestException.class,
-        "not a valid " +
-        "key=value pair (missing equal-sign) in Authorization header");
+        "IncompleteSignature");
```

Comment:
As per the commit message "Removes string match so change in AWS S3 error message doesn't cause the test to fail", the complete string matching is to be removed right?

Reply:
yes. They've changed the string, and rather than try and keep up with whatever error message comes back, I'll just remove the match. We had to do the same with some of the assumed role tests a few releases back when they changed the max life of a role token from 60 minutes to 12h.

Reply:
I though you said you will completely remove the string matching. But we are still trying to match "**IncompleteSignature**"

Reply:
sorry, I was confused. it's looking for the structured bit of the error message